lib/puppet/functions/wildfly/objectify.rb

```diff
@@ -0,0 +1,24 @@
+# Converts a LIST of STRINGs and OBJECTs into an OBJECT of OBJECTs.
+Puppet::Functions.create_function(:'wildfly::objectify') do
+  dispatch :objectify do
+    param 'Array[Variant[String, Hash[String, Any]]]', :input_list
+  end
```

Comment:
Please also add the desired return type. https://www.puppet.com/docs/puppet/7/functions_ruby_signatures#return-type-method

Reply:
okay. was wondering why puppet string says it returns any. so i will add a return type